### PR TITLE
New seed

### PR DESCRIPTION
**Pull request by jayk-u**

### CODE REVIEW

**Review by DumontNicolas — APPROVED** [review]
Yes I love those er tags

File: db/seeds.rb [hunk 1]
```diff
@@ -5,47 +5,86 @@
...
+  )
+  game.save!
+end
+
+p "Games generation completed"
+
 
-p "4 games have been added."
 p "And now, reservations..."
+stat = ["pending", "cancelled", "confirmed"]
```
Comment (DumontNicolas) [inline db/seeds.rb]: confirmed should be completed :)

Comment (jayk-u) [reply db/seeds.rb]: resolved

**Review by DumontNicolas — APPROVED** [review]
C'est du bon travail on en attends autant en prod

**Pull request merged**